.github/workflows/FormatCheck.yml

```
@@ -0,0 +1,17 @@
+name: Format suggestions
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags: '*'
+  pull_request:
+
+jobs:
+  format-suggestions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: julia-actions/julia-format@v3
```

```suggestion
      - uses: julia-actions/julia-format@v4
```
This should save you a dependabot PR in a couple of days :wink: